src/assets/articleAssets/tags.scss

```diff
@@ -0,0 +1,87 @@
+
+.tag-name {
+    background-color: grey;
+    color: white;
+    padding-left: 20px;
+    padding-right: 20px;
+    border-radius: 5px;
+    width: 100px;
+    margin-right: 10px;
+    background-color: rgba(119, 20, 137, 0.7);
+    border-color: rgba(119, 20, 137, 0.7);
+    padding-top: 4px;
+    padding-bottom: 5px;
+  }
+  
+  .tag-name:first-child {
+    margin-left: -40px;
+    margin-right: 50px;
+    
+  }
+
+  .tag-title {
+    margin:0 auto;
+    max-width:100%;
+    font-size:25px;
+    padding-left:43px;
+    padding-right: 20px;
+  }
+
+  
+
+  .tag-title-container{
+    column-count: 4;
+    column-gap: 1.5em;
+    width: 150px;
+    background-color: rgba(119, 20, 137, 0.7);
+    color:#fff;
+    padding-top:7px;
+    padding-bottom: 10px;
+    border-radius: 25px;
+    margin-left:0px;
+    margin-bottom:20px;
+    margin-top:30px;
+  }
+
+  
+  @media screen and (min-width:1200px) and (max-width: 1472px)  {
+    .tag-title-container {
+      margin-left:140px;
+      
+    }
+  }
+  @media screen and (max-width: 1200px)  {
+    .tag-title-container {
+      margin-left:100px;
+      
+    }
+  }
+
+  @media screen and (max-width: 600px)  {
+    .tag-title-container {
+      margin-left:55px;
+      
+    }
+  }
+
+  @media screen and (min-width:1925px) and (max-width: 2560px)  {
+    .tag-title-container {
+      margin-left:260px;
+    }
+  }
+
+  @media screen and (min-width:900px) and (max-width: 1053px)  {
+    .tag-title-container {
+      margin-left:100px;
+    }
+  }
+  @media screen and (min-width:772px) and (max-width: 923px)  {
+    .tag-title-container {
+      margin-left:80px;
+    }
+  }
+
+
+
+
+
```

Comment:
Remove empty lines of code, and leave one empty line at end of file, ensure to do this in all files